docker/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:latest as base
+
+LABEL maintainer="veandco"
+
+ENV DEBIAN_FRONTEND "noninteractive"
+
+RUN dpkg --add-architecture i386; \
+    apt-get update; \
+    apt-get install -y make cmake git gcc g++ libxml2-dev libssl-dev zlib1g-dev zlib1g-dev:i386 \
+        libgl-dev libgl-dev:i386  \
+        gcc-mipsel-linux-gnu g++-mipsel-linux-gnu  \
+        curl clang unzip wget gcc-arm-linux-gnueabi \
+        gcc-i686-linux-gnu gcc-mingw-w64 libxext-dev
+
+# Needed for compiler-rt in osxcross (https://github.com/tpoechtrager/osxcross/issues/214#issuecomment-595279668)
+RUN ln -sf /usr/bin/llvm-config-10 /usr/bin/llvm-config
+
+FROM base as android_ndk
+
+ARG ANDROID_NDK_VERSION="r24"
+ENV ANDROID_NDK_VERSION ${ANDROID_NDK_VERSION}
+
+RUN wget -O /opt/android-ndk.zip https://dl.google.com/android/repository/android-ndk-${ANDROID_NDK_VERSION}-linux.zip; \
+    unzip -q /opt/android-ndk.zip -d /opt; \
+    rm /opt/android-ndk.zip
+
+FROM android_ndk as macos
+
+WORKDIR /opt
+
+RUN git clone https://github.com/tpoechtrager/osxcross; \
+    wget -P osxcross/tarballs https://go-sdl2.veandco.sg-sin1.upcloudobjects.com/MacOSX10.14.sdk.tar.xz; \
```

Comment:
Just an FYI, the SDKs have been moved to the following URLs!
```
https://s3.veand.co/go-sdl2/MacOSX10.14.sdk.tar.xz
https://s3.veand.co/go-sdl2/MacOSX11.3.sdk.tar.xz
```

Reply:
Thanks. Hopefully I'll finally have time to finish this now during my vacation... :D